src/api/types.js

```diff
@@ -4,7 +4,8 @@
  * @typedef {Request<{ Server: { db: Db }, Params: FormBySlugInput }>} RequestFormBySlug
  * @typedef {Request<{ Server: { db: Db }, Params: FormByIdInput, Payload: FormDefinition }>} RequestFormDefinition
  * @typedef {Request<{ Server: { db: Db }, Payload: FormMetadataInput }>} RequestFormMetadataCreate
- * @typedef {Request<{Server: {db: Db}, Params: FormByIdInput, Payload: {force: boolean}}>} RequestRemoveFormById
+ * @typedef {Request<{ Server: { db: Db }, Params: FormByIdInput, Payload: {force: boolean}}>} RequestRemoveFormById
```

Comment:
`{ force: boolean }` minor spacing change to match others `@typedef`s

Reply:
done